src/code.sh

```
@@ -4,149 +4,260 @@
+        err "run_CollectVariantCallingMetrics was requested, but one or more of vcf or dbsnp_vcf are missing. Exiting..."
+        exit 1
+    fi
+
+    ## Setup 
+    dx-download-all-inputs
+
+    # Calculate 90% of memory size for java
+    MEM=$(head -n1 /proc/meminfo | awk '{print int($2*0.9)}')
+    MEM_IN_MB="$(("${MEM}"/1024))m"
+
+    tar zxvf "$fasta_index_path"
```

Shellcheck (0.10.0)

[warning] fasta_index_path is referenced but not assigned.

(SC2154)